vehicle_diag_smach/low_level_states/classify_components.py

```diff
@@ -240,9 +240,16 @@ def classify_with_torch_model(
         # )
 
         for i in range(len(var_attr_heatmaps)):
-            heatmap_id = self.instance_gen.extend_knowledge_graph_with_heatmap(
-                "XCM GradCAM", var_attr_heatmaps["var. attr. map " + str(i)].tolist()
+            heatmap_ids.append(
+                self.instance_gen.extend_knowledge_graph_with_heatmap(
+                    "XCM GradCAM variable attribution map", var_attr_heatmaps["var. attr. map " + str(i)].tolist()
+                )
             )
+        heatmap_ids.append(
+            self.instance_gen.extend_knowledge_graph_with_heatmap(
+                "XCM GradCAM time attribution map", time_attr_heatmaps["time attr. map " + str(0)].tolist()
```

Comment:
Why are you only adding the first of the time attribution maps? Ah, it's because they are always the same, right? We only assigned them to all the channels, but the values are always the same. I think it would be good to have a brief comment above the addition of the time attribution heatmap ID  to clarify.